### DISCUSSION TITLE
一些关于 PAI 的中文中介的截图

### DISCUSSION THREAD

**Discussion by eunicechauuu** [project-trans/basket-relay#8] [General]
网上传来的截图，关于某一位出了事却不理人的中介 😱 各位小心啊（名字打了码）

![001](https://github.com/project-trans/reports/assets/165674513/54ecd25d-9a35-4496-80b6-d5b696d235c1)
![002](https://github.com/project-trans/reports/assets/165674513/eb39adf8-a34e-46bf-8ffe-7e3af2e57f42)
![003](https://github.com/project-trans/reports/assets/165674513/add68191-3d0a-4746-9ad1-7bbf0616290a)
![004](https://github.com/project-trans/reports/assets/165674513/0617d9d8-fc05-4985-872c-b5a5f84c433a)
![005](https://github.com/project-trans/reports/assets/165674513/8b96e18b-3384-4c9f-88b2-13dddd1b161c)
![006](https://github.com/project-trans/reports/assets/165674513/f00ecfea-01c6-417a-9048-ab2fc7ca23c1)
![007](https://github.com/project-trans/reports/assets/165674513/6e98f404-386a-4532-a3e6-8ad1d6ed7eeb)
![008](https://github.com/project-trans/reports/assets/165674513/de770465-79ae-49f3-afce-83caaa82125d)

**Comment by Cryolitia** [comment]
您好，我们已经在相关页面添加了醒目警告，感谢您的反馈
![image](https://github.com/project-trans/reports/assets/23723294/5ca6194c-ee73-42b6-93b3-ee4800856f07)

related: https://github.com/project-trans/MtF-wiki/pull/1021

**Comment by response-pai-grs** [comment]
大家好，我是本图的主人公，很抱歉因为我个人的原因，让某些人用部分截图，影响对李萍的客观评价，接下来我想阐述真是情况⬇️

当天晚上我因为那一天的走动，然后导致下体出血，第一次经历这种情况是比较恐慌的，然后第一时间就联系了萍姐，萍姐说可能是一点小问题，让我不要恐慌，当时在开车。我其实抱着一些想找到一些方法的念头，去群里边去进行咨询。有没有人遇到过当时类似的情况，想给一些解决办法，然后也找了一些人帮忙，然后出现了一些下体的对话。因为自己当时第一次经历这种情况，内心比较慌张，所以就四处。打听，问一下原因等等等。

但是萍姐并没有说不帮我处理，首先他让我先不要恐慌，按照他说的一些动作进行调整，然后也给我发了一些其他人的案例，让我不要太过于担心。最后也说了，要不就最后一个方案，就是如果真的有问题的话，也可以联系医院或者是第二天问一下医生。

总体来说我在我在泰国的这段时间，萍姐都是比较负责任的，比如买一些东西，都会第一时间给我们送到，所以我不希望一些不好的言论给她。

_**然后我想综合的评价一下，李萍是一个比较负责任的中介。我希望大家再去选择的时候，不要过多的听外界的声音。自己去感受这个人。当时刚去泰国的时候，刚接触的时候也是有一点不放心，因为确实一直有一些好的言论对他造成一些影响，但是当我综合下来接触这个人的时候，我真的觉得李萍还是比较负责的，希望大家勇敢追梦。**_
【附带一些图片⬇️】萍姐很负责

![0](https://github.com/project-trans/reports/assets/169511382/289fdd99-288d-41f4-8e6e-ff9e67c3c832)
![1](https://github.com/project-trans/reports/assets/169511382/5224d46a-4bf8-4d3d-9dad-5c160048b8d6)
![2](https://github.com/project-trans/reports/assets/169511382/217732e3-224b-4357-a55b-71e23b10fdcd)
![3](https://github.com/project-trans/reports/assets/169511382/fb666d0d-454f-4b40-a92e-3f6c10c8e5e5)
![4](https://github.com/project-trans/reports/assets/169511382/89e48a3b-bad6-4c15-9b90-0253cbb6be98)
![5](https://github.com/project-trans/reports/assets/169511382/bdf5f412-a993-43a8-b0fb-3a47506f7d4b)
![6](https://github.com/project-trans/reports/assets/169511382/aac561e9-2c6f-41b9-90e8-de85010c8687)
![7](https://github.com/project-trans/reports/assets/169511382/05b8c69e-3d46-4481-81e4-68eea268e3a1)
![8(https://github.com/project-trans/reports/assets/169511382/6e79ed08-7aa4-4eda-a67c-cfde9b4e99cf)
![9](https://github.com/project-trans/reports/assets/169511382/fb7699d7-b99a-4f46-a505-15ea6fba03b5)
